src/pages/Dashboard.css

```
@@ -1,3 +1,12 @@
 h1 {
     text-align: center;
 }
+
+.cards{
```

```suggestion
.cards {
```

ez mindenhol legyen meg